internal/logging/logger.go

```diff
@@ -91,7 +91,18 @@ func Middleware() gin.HandlerFunc {
 
 		c.Next()
 
-		log.Info().
+		levelName := zerolog.InfoLevel
+		if c.IsAborted() {
+			levelName = zerolog.ErrorLevel
+		}
```

Comment:
Because of how we implemented the `UIProxy` every request through the proxy is aborted. So this might not behave correctly in all cases. Maybe be better to check the length of `c.Errors`.

Reply:
I don't see this. UI requests are logged with info regardless if they're served from the static fs or proxied to an external server

Reply:
Good catch. Checking `c.Errors` is probably the right solution